src/test/java/com/qunite/api/web/QueueControllerTest.java

```diff
@@ -164,14 +138,14 @@ void retrieveQueueManagers() throws Exception {
 
     given(queueService.findById(anyLong())).willReturn(Optional.of(queue));
     var resultActions = mockMvc.perform(get(url + "/" + queue.getId() + "/managers")
-        .accept(MediaTypes.JSON_API_VALUE));
+        .accept(MediaType.APPLICATION_JSON_VALUE));
     resultActions.andExpect(status().isOk())
-        .andExpect(jsonPath("data", hasSize(queue.getManagers().size())))
-        .andExpect(header().string(HttpHeaders.CONTENT_TYPE, MediaTypes.JSON_API_VALUE))
+        .andExpect(jsonPath("$", hasSize(queue.getManagers().size())))
+        .andExpect(header().string(HttpHeaders.CONTENT_TYPE, MediaType.APPLICATION_JSON_VALUE))
         .andExpect(content().string(allOf(
-            containsString("\"id\":\"1\""),
-            containsString("\"id\":\"2\""),
-            containsString("\"id\":\"3\""))));
+            containsString("\"id\":1,"),
+            containsString("\"id\":2,"),
+            containsString("\"id\":3,"))));
```

Comment:
There are also some commas here, so if you need to check for specific values, use other methods.